controllers/cluster_controller_test.go

```diff
@@ -279,6 +279,53 @@ var _ = Describe(string(fdbtypes.ProcessClassClusterController), func() {
 			})
 		})
 
+		Context("when buggifying an empty fdbmonitor conf", func() {
+			BeforeEach(func() {
+				cluster.Spec.Buggify.EmptyMonitorConf = true
+				err = k8sClient.Update(context.TODO(), cluster)
+				Expect(err).NotTo(HaveOccurred())
+
+				// The buggify config causes the reconciliation loop to never finish with the mock admin client.
+				shouldCompleteReconciliation = false
+			})
+
+			It("should update conf and bounce processes", func() {
+				pods := &corev1.PodList{}
+				err = k8sClient.List(context.TODO(), pods, getListOptions(cluster)...)
+				Expect(len(pods.Items)).To(Equal(len(originalPods.Items)))
+				sortPodsByID(pods)
+
+				cm, _ := GetConfigMap(cluster)
+
+				for _, file := range cm.Data {
+					Expect(file).To(Not(ContainSubstring("fdbserver")))
+				}
+
+				adminClient, err := newMockAdminClientUncast(cluster, k8sClient)
+				Expect(err).NotTo(HaveOccurred())
+				Expect(adminClient).NotTo(BeNil())
+
+				// Converted the killed servers into a set since they may have been killed more than once during
+				// reconciliation.
+				killedSet := map[string]struct{}{}
```

Comment:
I like that you use the empty struct here :) normally I try to prevent that and choose to use a `bool`.

Reply:
I like the empty struct implementation I only figured out that it's a little bit confusing for people that are not used to go.